features/admin.actions.v1/pages/actions.tsx

```diff
@@ -265,6 +266,51 @@ export const ActionTypesListingPage: FunctionComponent<ActionTypesListingPageInt
             } ];
     };
 
+
+    /**
+     * This function returns loading placeholders.
+     */
+    const renderLoadingPlaceholder = (): ReactElement => {
+        const placeholders: ReactElement[] = [];
+        const cardsPerRow: number[] = [ 3, 1 ];
+
+        for (let rowIndex: number = 0; rowIndex < cardsPerRow.length; rowIndex++) {
+            const cardsInRow: number = cardsPerRow[ rowIndex ];
+            const cards: ReactElement[] = [];
+
+            for (let columnIndex: number = 0; columnIndex < cardsInRow; columnIndex++) {
+                cards.push(
+                    <div data-componentid={ `${ _componentId }-loading-card` }>
```

Comment:
why do we need this div, can't we directly use the Box for this ?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/6764/commits/cf1bd171b56be2d259b3fe7dbf5679164eb9a7cf